Class2

ahrenswett opened the pull request.

ahrenswett committed:
Problems
recycler view not rendering. it seems that MainActivity is not calling class adapter.
DB is working and I am able to send / receive data from it.

ahrenswett committed:
Fixing recycler view

ahrenswett committed:
Additions/Challenges

ahrenswett committed:
Added Student class Dao and other supporting features

arybin commented on app/src/main/java/com/ahrenswett/classattendance/ClassAdapter.java:
```diff
@@ -26,30 +28,38 @@
 
 
+    private static void onBind(ClassViewHolder holder, Class classAtPosition){
+        holder.myClass = classAtPosition;
```
nit: consider pushing this logic down into the `ClassViewHolder`

ahrenswett committed:
Fixes and Improvements
In main activity when inflating the PopUpWindow I was getting a null object because I was refering to the current view not the view i was inflating to
Added an abstract class BaseViewHolder to ClassAdapter to extend the recycler view this will allow multiple ViewHolders to be made.

ahrenswett committed:
Added Strings to the string XML file for accessability
